demisto_sdk/commands/common/docker_helper.py

```diff
@@ -467,25 +483,49 @@ def _get_python_version_from_env(env: List[str]) -> Version:
 
 
 @functools.lru_cache
-def get_python_version(image: Optional[str]) -> Version:
-    log_prompt = f"Get python version from image {image}"
-    logger.debug(f"{log_prompt} - Start")
-    if not image or "pwsh" in image or "powershell" in image:
+def get_python_version(image: Optional[str]):
+    """
+    Get the python version of a docker image if exist.
+
+    Args:
+        image (str): the docker image
+
+    Returns:
+        Version: Python version X.Y (3.7, 3.6, ..)
+    """
+    logger.debug(f"Get python version from image {image=}")
+
+    if not image:
         # When no docker_image is specified, we use the default python version which is Python 2.7.18
         logger.debug(
             f"No docker image specified or a powershell image, using default python version: {DEFAULT_PYTHON2_VERSION}"
```

Comment:
when will we have `image == None` and when will `"pwsh" in image or "powershell" in image` be True? why in the first case we return `Version(DEFAULT_PYTHON2_VERSION)` and in the second one `None`?

Reply:
`image = None` is for cases where we have scripts/integrations that are based on javascript, @ilaner do you remember why we set the `DEFAULT_PYTHON2_VERSION` here? 

I think its related to the platform if i recall

Reply:
> `image = None` is for cases where we have scripts/integrations that are based on javascript, hence they do not use any docker image, @ilaner do you remember why we set the `DEFAULT_PYTHON2_VERSION` here?
> 
> I think its related to the platform if i recall

If the docker image is missing, the server will run the code in docker image which based on python 2 image.